argschema/fields/files.py

```diff
@@ -41,23 +60,26 @@ def _validate(self, value):
             path = os.path.dirname(value)
         except Exception as e: # pragma: no cover 
             raise mm.ValidationError("%s cannot be os.path.dirname-ed" % value) # pragma: no cover 
-        try:
-            with tempfile.TemporaryFile(mode='w', dir=path) as tfile:
-                tfile.write('0')
-        except Exception as e:
-            if isinstance(e, OSError):
-                if e.errno == errno.ENOENT:
-                    raise mm.ValidationError(
-                        "%s is not in a directory that exists" % value)
-                elif e.errno == errno.EACCES:
-                    raise mm.ValidationError(
-                        "%s does not appear you can write to path" % value)
-                else:
-                    raise mm.ValidationError(
-                        "Unknown OSError: {}".format(e.message))
-            else:
+        validate_outpath(path)
+
+class OutputDir(mm.fields.Str):
```

Comment:
I think Gayathri was interested in including setting the mode when creating directories to, e.g. `0o775`.

Reply:
added along with a test